layers/meta-balena-beaglebone/recipes-bsp/u-boot/u-boot-bb.org_%.bbappend

```diff
@@ -0,0 +1,25 @@
+FILESEXTRAPATHS:append := ":${THISDIR}/${PN}"
+
+UBOOT_KCONFIG_SUPPORT = "1"
```

Comment:
This is not needed anymore, can be removed

Reply:
What exactly in there is not needed anymore? My guess is `UBOOT_KCONFIG_SUPPORT = "1"` but being extra safe asking back :)

And by the way: why would that be?

Reply:
Yes, `UBOOT_KCONFIG_SUPPORT = "1"` is redundant nowadays because it is automatically added when you inherit resin-u-boot providing you are not using a really really old u-boot.

Reply:
Thanks for the feedback! I see you approved this PR, want me to add a new one for this once this one is in? (assuming nothing else pops out)

Reply:
If you want to

Reply:
https://github.com/balena-os/balena-beaglebone/pull/674